sfepy/discrete/fem/fields_base.py

```diff
@@ -629,7 +702,8 @@ def restore_substituted(self, vec):
         vec = vec.reshape((self.n_nod, self.n_components)).copy()
         evec = vec[self.econn]
 
-        vec[self.econn0] = nm.einsum('cji,cjk->cik', self.basis_transform, evec)
+        vec[self.econn0] = nm.einsum('cji,cjk->cik', self.basis_transform,
+                                     evec)
```

Comment:
Maybe turn on optimization too? But for two operands it IMO does not matter.

Reply:
`optimization=True` costs nothing and does no harm